packages/worker/src/index.js

```diff
@@ -14,6 +15,14 @@ const redis = require("./utilities/redis")
 
 const app = new Koa()
 
+// TODO: Remove this when using envoy / nginx
+const accountPortalCors = {
+  origin: "http://localhost:3001",
+  credentials: true,
+}
+
+app.use(cors(accountPortalCors))
```

Comment:
Since this is just for dev, we should use the `isDev()` check from `environment.js` to avoid the middleware being attached in prod.

Reply:
I think it will have to remain in some form, at least in the current implementation so that the account-portal web will be able to talk to budibase ui. 

I'll update this with environment specific values for now